Dockerfile

```diff
@@ -83,10 +83,147 @@ RUN mkdir -p /opt/flexbar/tmp \
 #Toil#
 ######
 RUN pip install --upgrade pip \
-    && pip install toil[cwl]==3.6.0 \
+    && pip install toil[cwl]==3.12.0 \
```

Comment:
We'd love to upgrade to the newest version of `toil`, but there are some interesting issues it raises with changes it makes to the LSF support.  Did you encounter any issues running a pipeline with the upgraded version?

Reply:
I did not actually test it via LSF - only locally. Will run that test now.  My goal was to resolve differences in validation status between this container and the version travis was running.

Reply:
New version of toil seems to work smoothly, both locally and via LSF